designs/administrator/4.6/column-management/index.md

```diff
@@ -0,0 +1,53 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Column Management
+
+## Behavior
+Column management will live in the toolbar as a column icon on the Pods page on desktop. It will be hidden on mobile. Clicking on the icon will open a modal that lists all column names with checkboxes.
```

Comment:
```suggestion
Column management will live in the toolbar as a column icon on the relevant resource pages on desktop. It will be hidden on mobile. Clicking on the icon will open a modal that lists all column names with checkboxes.
```

Reply:
@gdoyle1 is there a tooltip for the column management icon?

Reply:
@serenamarie125 There isn't right now. Do you think that'd be useful? PatternFly has also used a link button that says `Manage columns` in the toolbar although since we have limited space in the toolbar, it seems like the column icon button would be a better choice here